wagtail/admin/templates/wagtailadmin/home/locked_pages.html

```diff
@@ -49,7 +49,7 @@ <h2 class="title-wrapper">{% trans "Your locked pages" %}</h2>
                                 </ul>
                             </td>
                             <td>{# Deliberately empty #}</td>
-                            <td valign="top"><div class="human-readable-date" title="{{ page.locked_at|date:"DATETIME_FORMAT" }}">{% blocktrans trimmed with time_period=page.locked_at|timesince_simple %}{{ time_period }}{% endblocktrans %}</div></td>
+                            <td valign="top">{% human_readable_date date=page.locked_at %}</td>
```

Comment:
As the date is the first Arg and required - it might be simpler to update all of these to just pass a date in as the first Arg. 
```suggestion
                            <td valign="top">{% human_readable_date page.locked_at %}</td>
```